### ISSUE TITLE
the results of debertav3 small on the mnli task

### ISSUE THREAD

**Issue by nbcc** [microsoft/DeBERTa#72]
The results of debertav3 small on the mnli validation set, in the paper, reported as 88.2/87.9, look different from those reported in open source: https://huggingface.co/mrm8488/deberta-v3-small-finetuned-mnli (reported as 87.46)

<img width="809" alt="图片" src="https://user-images.githubusercontent.com/864331/142985570-715f92c9-a0d3-4ab5-8a38-0a4d7c337dae.png">

**Comment by MoritzLaurer** [comment]
that's most likely because for the paper Microsoft did extensive hyperparameter tuning, while mrm8488 didn't before uploading it to the hub

**Comment by BigBird01** [comment]
Please follow our [script to set the hyper-parameters](https://github.com/microsoft/DeBERTa/blob/master/experiments/glue/mnli.sh), e.g. `mnli.sh deberta-v3-small`
